OctopusDSC/OctopusDSCHelpers.ps1

```diff
@@ -158,41 +158,45 @@ Function Get-MaskedOutput
     [CmdletBinding()]
     param($arguments)
 
-    $reg = [System.Text.RegularExpressions.RegEx]::new("--masterkey|--password|--license",
```

Comment:
We felt it was prudent to improve the logic around the masking function. Note the edge case clause (the else block).